openshift/scripts/oc-dump.sh

```diff
@@ -92,8 +92,9 @@ SAVE_PATH=$( dirname ${SAVE_TO} )
 mkdir -p ${SAVE_PATH}
 oc exec ${POD_DB} -n ${PROJECT} -- /bin/bash -c '\
   pg_dump -U ${POSTGRESQL_USER} -d ${POSTGRESQL_DATABASE} -Fc -f /tmp/'${SAVE_FILE}' \
-	--no-privileges --no-tablespaces --schema=public \
+	--no-privileges --no-tablespaces --schema=public --exclude-table=spatial_ref_sys \
```

Comment:
@DerekRoberts With the new PostGIS extension, we now have a 'system/DB wide table' in the PUBLIC schema.  We do NO want to dump/restore that with the GWELLS objects, so I'm excluding it from the dump.